aten/src/ATen/native/cuda/UnaryOpsKernel.cu

```diff
@@ -40,6 +40,14 @@ void ceil_kernel_cuda(TensorIterator& iter) {
   });
 }
 
+void round_kernel_cuda(TensorIterator& iter) {
+  AT_DISPATCH_FLOATING_TYPES_AND_HALF(iter.dtype(), "round_cuda", [&]() {
+    gpu_kernel(iter, []GPU_LAMBDA(scalar_t a) -> scalar_t {
+      return std::round(a);
```

Comment:
See #17443 we had reasons to use `nearbyint`

Reply:
I see, I have made the change and add a comment.